product_pricelist_direct_print/views/report_product_pricelist.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Carlos Dauden <carlos.dauden@tecnativa.com>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+
+<template id="report_product_pricelist_document">
+    <t t-call="web.html_container">
+        <t t-call="web.external_layout">
+            <t t-set="pricelist" t-value="o.get_pricelist_to_print()"/>
+            <div class="page">
+                <h2>Price List</h2>
```

Comment:
@pedrobaeza All changes in #581 are implemented except this line that was deleted.
But, I am not sure that it should be deleted